pybop/__init__.py

```diff
@@ -1,13 +1,13 @@
 #
-# Root of the pybop module.
+# Root of the PyBOP module.
 # Provides access to all shared functionality (models, solvers, etc.).
 #
 # This file is adapted from Pints
 # (see https://github.com/pints-team/pints)
 #
 
 import sys
-import os
+from os import path
```

Comment:
Maybe don't need to change, depends on whether we need any functionality from os in the future.